sdk/ai/azure-ai-agents/samples/agents_streaming/sample_agents_stream_eventhandler_with_functions.py

```diff
@@ -40,10 +40,11 @@
 )
 from azure.identity import DefaultAzureCredential
 
-current_path = os.path.dirname(__file__)
-root_path = os.path.abspath(os.path.join(current_path, os.pardir, os.pardir))
-if root_path not in sys.path:
-    sys.path.insert(0, root_path)
+# Add package directory to sys.path to import user_functions
+current_dir = os.path.dirname(os.path.abspath(__file__))
```

Comment:
I wonder if this could be under utility code module that gets used commonly in samples?

Reply:
sounds good